Client/Frontend/Home/HomepageViewModelProtocol.swift

```diff
@@ -45,8 +45,6 @@ extension HomepageViewModelProtocol {
     func updatePrivacyConcernedSection(isPrivate: Bool) {}
 
     func refreshData(for traitCollection: UITraitCollection,
-                     isPortrait: Bool = UIWindow.isPortrait,
-                     device: UIUserInterfaceIdiom = UIDevice.current.userInterfaceIdiom) {
-        refreshData(for: traitCollection, isPortrait: isPortrait, device: device)
-    }
+                     isPortrait: Bool,
```

Comment:
This is actually how the code looks at the moment on the Firefox side, this default implementation actually generated an endless loop since it called itself 😅